plugin/src/test/java/org/opensearch/ml/tools/CatIndexToolIT.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.tools;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import org.junit.Before;
+import org.opensearch.client.Response;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.ml.rest.RestBaseAgentToolsIT;
+import org.opensearch.ml.utils.TestHelper;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class CatIndexToolIT extends RestBaseAgentToolsIT {
+    private String agentId;
+    private final String question = "{\"parameters\":{\"question\":\"please help list all the index status in the current cluster?\"}}";
+
+    @Before
+    public void setUpCluster() throws Exception {
+        registerCatIndexFlowAgent();
+    }
+
+    private List<String> createIndices(int count) throws IOException {
```

Comment:
This is called multiple times in this test. How are we cleaning up the indices between tests?  Is that in the superclass?  Otherwise creating 10 will duplicate the first 10 of the 101 given the current setup.

Reply:
please refer to: https://github.com/opensearch-project/ml-commons/pull/3243#discussion_r1868825815